tools/integration-test/src/tests/fee/forward_relayer.rs

```diff
@@ -0,0 +1,136 @@
+use ibc_relayer_types::core::ics04_channel::version::Version;
```

Comment:
```suggestion
//! Tests the relayer's support of fee transfer functionality. 
//! 
//! The test performs an `ibc_token_transfer_with_fee` between `chain_driver_a` 
//! (the source chain) and `chain_driver_b` (the destination chain). Before it can perform
//! this operation, it first queries for the `counterparty_payee`'s address so that the
//! fees can be sent. It then checks to make sure all the appropriate fees are paid 
//! out to the correct parties involved in the transaction.

use ibc_relayer_types::core::ics04_channel::version::Version;
```

Reply:
As discussed, the part which I feel should be rephrased is with the query of the counterparty payee, something along the lines of:

//! (the source chain) and `chain_driver_b` (the destination chain). Before it can perform
//! this operation, it first checks that no `counterparty_payee` has been registered, and
//! then registers the `relayer_a`'s address as the `counterparty_payee` so that the fees
//! will be sent to it. After the transfer, it checks to make sure all the appropriate fees are
//! paid out to the correct parties involved in the transaction.